Dockerfile

```diff
@@ -34,7 +34,9 @@ RUN apt-get update && \
     libxtrx-dev \
     pkg-config \
     wget \
-    python3-six
+    python3-six \
+    openssh-client \
+    ffmpeg
```

Comment:
I think these packages need to go on line 53 of this file in the final Docker image, rather than this step which is just the intermediate build image.